README.md

```diff
@@ -24,10 +24,12 @@ Unzipped the full parquet data will be approximately 70GB. Each (compressed) chu
 
 ### New Contributor Tips
 
-- A valid and much appreciated contribution is to write any of your learnings, be it by reading research papers or through your interaction with the community on gitter and submitting a Pull Request to the repository. You can submit the PR to the README on the main page or the analysis folder README.
+- Your core focus should be an analysis / contribution riffing off one of the research-question issues. Having said that any open-source project, even the non-traditional ones, need constant improvements from lots of fresh eyes. A valid and much appreciated contribution is to write any of your learnings, be it by reading research papers or through your interaction with the community on gitter and submitting a Pull Request (PR) to the repository. You can submit the PR to the README on the main page or the analysis folder README.
 
 - This is not a one issue per person repo. All the questions are very open ended and different people may find very different and complementary things when looking at a question.
 
+- For datasets : Dataset is not to be fit into memory it will always be working with spark or dask to process an analysis out-of-core. 
```

Comment:
This is great. Small english tweaks. Suggest: The dataset is very large. Even samples are unlikely to fit into memory. Working with this dataset will typically require using Dask (http://dask.pydata.org/), Spark (http://spark.apache.org/) or similar tools to enable parallelized / out-of-core / distributed processing.